pkgs/servers/trezord/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchgit, curl, cmake, boost, gcc5, protobuf, pkgconfig, jsoncpp
+, libusb1, libmicrohttpd
+}:
+
+let
+  version = "1.2.0";
+in
+
+stdenv.mkDerivation rec {
+  name = "trezord-${version}";
+
+  src = fetchgit {
```

Comment:
`fetchgit` should only be necessary if you need submodules. Can you elaborate on the build error?